python/lsst/cell_coadds/_GridContainer.cc

```diff
@@ -85,6 +104,23 @@ void wrapGridContainer(utils::python::WrapperCollection& wrappers) {
             cls.def("__len__", &GridContainer<py::object>::size);
             cls.def_property_readonly("first", &GridContainer<py::object>::get_first);
             cls.def_property_readonly("last", &GridContainer<py::object>::get_last);
+            cls.def(
+                "get",
+                [](GridContainer<py::object> const& self, int x, int y) -> py::object {
+                    GridIndex index{x, y};
+                    check_index(index, self);
+                    return self[index];
+                },
+                py::kw_only(),
+                "x"_a,
+                "y"_a);
+            cls.def(
```

Comment:
For `afwImage` objects, we could pass in slices within square brackets and get a view (I guess?) of the image. I believe that's not possible on these images? I'm thinking of supporting a syntax like
```python
container.get(x=1:5, y=4:7)
```
or at least like
```python
container.get(x=slice(1,5), y=slice(4,7))
```

I suppose we could do this by first constructing a `bbox` and perhaps using the `subset_overlapping` method.

Reply:
The first syntax isn't legal, unfortunately.  I didn't think the latter was worth supporting as a shortcut for `subset_overlapping.  That's in part because I didn't want to imply too much that this behaves like Python sequence indexing: because these are absolute indices over the whole patch, they don't necessarily start at 0 for any particular `GridContainer`, and that makes it impossible to support negative indexes in a non-confusing way.